server/src/main/scala/org/http4s/server/middleware/authentication/BasicAuth.scala

```diff
@@ -46,7 +46,7 @@ object BasicAuth {
         case Some(authInfo) =>
           Right(AuthedRequest(authInfo, req))
         case None =>
-          Left(Challenge("Basic", realm, Map.empty))
+          Left(Challenge("Basic", realm, Map("charset" -> "UTF-8")))
```

Comment:
This parameter map is going to appear a lot in servers that support this, so I'd put it in a val.

Reply:
fixed